src/DataMigration/DataMigration/help/New-AzDataMigrationAzureActiveDirectoryApp.md

```diff
@@ -0,0 +1,94 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.DataMigration.dll-Help.xml
+Module Name: Az.DataMigration
+online version: https://docs.microsoft.com/en-us/powershell/module/az.datamigration/New-AzDataMigrationAzureActiveDirectoryApp
+schema: 2.0.0
+---
+
+# New-AzDataMigrationAzureActiveDirectoryApp
+
+## SYNOPSIS
+Create a new instance DataMigration Azure ActiveDirectory Application details.
+
+## SYNTAX
+
+```
+New-AzDataMigrationAzureActiveDirectoryApp -ApplicationId <String> -AppKey <SecureString>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create a new instance DataMigration Azure ActiveDirectory Application details.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> $secpasswd = ConvertTo-SecureString "Your Secret Key Here" -AsPlainText -Force
+New-AzDmsAadApp -ApplicationId "Your AppId/Service Principal ID here" -AppKey $secpasswd
```

Comment:
@sichoudh nit: add `PS C:\>` before the command call on this line for consistency

Also, please add the corresponding output from this cmdlet being run so users know what is returned when calling this cmdlet.